eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -47,9 +52,13 @@ steps:
 # exist, create it
 - ${{ if eq(parameters.DailyDocsBuild, 'true') }}:
   - template: /eng/common/pipelines/templates/steps/set-daily-docs-branch-name.yml
-
+  
   - pwsh: |
       $ErrorActionPreference = "Continue"
+      # Add remote fork repo
```

Comment:
Why do we need to add this remote here as opposed to just always cloning via the azure-sdk for? In other words why cann't we just pass azure-sdk for the TargetDocRepoOwner parameter?